x-pack/legacy/plugins/siem/server/lib/detection_engine/rules/prepackaged_rules/windows_modification_of_boot_config.json

```diff
@@ -1,36 +1,35 @@
-{
-  "description": "Identifies use of bcdedit.exe to delete boot configuration data. This tactic is sometimes used as by malware or an attacker as a destructive technique.",
-  "index": [
-    "winlogbeat-*"
-  ],
-  "language": "kuery",
-  "max_signals": 33,
```

Comment:
Same as above, shouldn't this result in a version bump?

Reply:
we will re-check those in the next build.